Frenchlocaleupdates

adarshamrit opened the pull request:
Closes #497

- Translated all documentation files from English to French and placed them in docs.
- Updated all internal Markdown links in the French docs to point to the translated versions.
- Copied required assets to actifs and updated asset links in the French docs.
- Ensured folder structure and filenames match the originals for easy navigation.
- No changes were made to the original English documentation or codebase.

adarshamrit committed:
Updated French localizations

adarshamrit committed:
Update logo in French localization

- Changed the logo file reference in README.md from `logo-and-name.svg` to `logo.svg`.
- Added new SVG logo file `logo.svg` to the French localization assets directory.

goatshriek commented:
Thanks very much for putting this together! There is a lot to review here, so it will take me a bit to go through it. I will post a review once I have, with any changes requested then.

goatshriek commented on l10n/fr-fr/README.md:
```diff
@@ -0,0 +1,256 @@
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-v2.1-ff69b4.svg)](https://github.com/goatshriek/stumpless/blob/latest/docs/CODE_OF_CONDUCT.md)
+
+
+[한국어](./l10n/ko-kr/README.md) | [简体中文](./l10n/zh-cn/自述.md)
```
Please add a link to the English README page here.

goatshriek commented on l10n/fr-fr/README.md:
````diff
@@ -0,0 +1,256 @@
+sudo cmake --install .
+```
+
+Consultez les [instructions d'installation](INSTALL.md) pour plus de détails sur
````
The link to INSTALL is broken here - you could simply remove the link, or link back to the English version as a fallback, either is fine.

goatshriek commented on l10n/fr-fr/README.md:
```diff
@@ -0,0 +1,256 @@
+### Encore plus d'exemples
+Pour des exemples plus détaillés des scénarios ci-dessus, l'utilisation de types de cibles spécifiques,
+comment gérer des structures de messages plus complexes, et plus encore, consultez les
+[exemples](docs/examples). Ceux-ci incluent des fichiers de code annotés à compiler,
```
This link is broken as well - again, either simply remove the link or link back to the main copy as a fallback.

goatshriek commented on l10n/fr-fr/docs/CODE_OF_CONDUCT.md:
```diff
@@ -0,0 +1,39 @@
+# Code de Conduite Stumpless
```
For this file, you can simply include the translation provided by the contributor covenant folks, which is complete. You can find it [here](https://www.contributor-covenant.org/fr/version/2/1/code_of_conduct/code_of_conduct.md).

goatshriek commented on l10n/fr-fr/docs/CONTRIBUTING.md:
```diff
@@ -0,0 +1,69 @@
+ * Si un commit résout ou est lié à une issue particulière, mentionnez-la dans le corps du commit.
+
+### **Ce serait bien si...**
+Si vous avez une nouvelle fonctionnalité à proposer, ouvrez une issue sur le projet github en utilisant le modèle "Feature Request". Si cela correspond au projet, vous pouvez commencer à travailler sur un fork et soumettre une pull request pour que nous puissions suivre l'avancement et l'intégrer une fois terminée. Sinon, nous l'ajouterons à la [feuille de route](roadmap.md) et l'implémenterons s ...
```
The roadmap link is broken here.

goatshriek reviewed (changes requested):
This is a lot of translation work, thanks very much for putting it together!

There are some minor edits requested, mostly around broken links. The only other request I have is to update the main README to link to the French translation README as is already done for Korean and Chinese, so that folks can easily benefit from this hard work!

adarshamrit committed:
Fixed the issues according to the feedback in the PR.

Also added the link to French Readme.md in the English version navbar beside Korean and Chinese.

adarshamrit commented:
This is my first Open Source Contribution. Thank you very much for the feedback to make my work detailed and accurate. Please leave feedback if further changes are needed.

goatshriek reviewed (approved):
Excellent work, thanks once again for putting in such a large translation contribution!
